go-controller/pkg/node/management-port.go

```diff
@@ -33,7 +33,7 @@ type ManagementPort interface {
 }
 
 // NewManagementPorts creates a new ManagementPorts
-func NewManagementPorts(nodeName string, hostSubnets []*net.IPNet) []ManagementPort {
+func NewManagementPorts(nodeName string, hostSubnets []*net.IPNet, netdevName, rep string) []ManagementPort {
```

Comment:
`netdevName` should have a type `string`?

Reply:
So it is